### PR TITLE
Small fixes to keep this working on new systems

### PR DESCRIPTION
**Pull request by Flova**
Some minor fixes for new python versions.

### CODE REVIEW

File: sshsysmon/templates/htmldash.hb [hunk 1]
```diff
@@ -80,7 +80,7 @@ Colors from: http://clrs.cc/
 			<div class="container-fluid">
 				<div class="row">
 					{{!-- Alarms --}}
-					<div class="col-xs-4 inspector">
+					<div class="col-xs-12 col-md-4 inspector">
```
Comment (zix99) [inline sshsysmon/templates/htmldash.hb:83]: What's this change for?

File: sshsysmon/inspectors/gpu.py [hunk 2]
```diff
@@ -0,0 +1,52 @@
+import logging
```
Comment (zix99) [inline sshsysmon/inspectors/gpu.py:1]: Looks like a few things are imported aren't used. Can you clean up if able?

**Review by zix99 — COMMENTED** [review]
Hey, thanks for the contribution!  Happy to merge it after a few minor changes.

I haven't maintained this in a while, so am not set up to check all the changes but they look safe.  Have you gotten a chance to test the new ssh paramiko version to make sure there are no regressions?

Thanks!

File: sshsysmon/inspectors/gpu.py [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+Constructor:
+    device: The gpu id (eg 0 or 1) (Default: 0)
+
+Metrics:
+    size: ByteSize of the device
+    used: ByteSize of used memory
+    available: ByteSize of the available memory
+    percentage_full: The integer percentage of how full the memory is
+"""
+class GPUMemory(Inspector):
```
[the remark this answers is not in the document]
Comment (Flova) [reply sshsysmon/inspectors/gpu.py:19]: That was honestly a mistake. I accidentally pushed [a42c4e0](https://github.com/zix99/sshsysmon/pull/14/commits/a42c4e08fff6215aa97473b01c16e372a06a15bf) on this branch. But we can include it into the PR if it is okay for you.

[the remark this answers is not in the document]
Comment (Flova) [reply sshsysmon/inspectors/gpu.py:19]: I would rename it or add a `vendor` parameter that raises a non-implemented error for AMD. I would build it generic for AMD and Nvidia, but I only have access to Nvidia GPUs.

File: sshsysmon/templates/htmldash.hb (hunk 1 above)
Comment (Flova) [reply sshsysmon/templates/htmldash.hb:83]: This was some formatting fix. The columns started overlapping on mobile devices and this gives each chart the full width if a tiny display is used.

File: sshsysmon/inspectors/gpu.py (hunk 2 above)
Comment (Flova) [reply sshsysmon/inspectors/gpu.py:1]: Sure

File: sshsysmon/inspectors/gpu.py (hunk 3 above)
[the remark this answers is not in the document]
Comment (zix99) [reply sshsysmon/inspectors/gpu.py:19]: Yeah, that works too!  I think it's a good change, even if it was accidental :)

[the remark this answers is not in the document]
Comment (Flova) [reply sshsysmon/inspectors/gpu.py:18]: Wow you answer really fast ^^

[the remark this answers is not in the document]
Comment (Flova) [reply sshsysmon/inspectors/gpu.py:18]: I implemented it in [8abd3c4](https://github.com/zix99/sshsysmon/pull/14/commits/8abd3c4b0018a75055aa78ea62a4e8b419b5f64e)

**Comment by zix99** [comment]
Great works, thanks!

**Pull request merged**